lammps-tutorials.tex

```diff
@@ -181,257 +306,573 @@ \section{Prerequisites}
+all atoms and then update the positions according to a selected
+algorithm so that the potential energy will decrease.  By default,
+LAMMPS uses the conjugate gradient (CG) algorithm
+\cite{hestenes1952methods}.  The simulation will stop as soon at the
+minimizer algorithm cannot find a way to to lower the potential energy.
+Except for trivial systems, minimization algorithms will find the
+``next'' minimum, which may not be the global minimum.
+
+Now you can run the simulation by selecting \textit{Run LAMMPS from Editor
+  Buffer} from the \textit{Run} menu, or by using the
+\texttt{Ctrl-Enter} keyboard shortcut or by clicking on the green ``Run
+LAMMPS'' button (second from left) in the status bar.  The simulation
+can be stopped at any time by using \textit{Stop LAMMPS} from the
+\textit{Run} menu, the \texttt{Ctrl-/} keyboard shortcut or by clicking
+on the red stop button in the status bar.  A new run will always
+``forget'' the current system and start over from the top.
+
+\begin{figure}
+\centering
+\includegraphics[width=0.45\linewidth]{chart-1}
+\includegraphics[width=0.45\linewidth]{output-1}
+\caption{\textit{Chart} (left) and \textit{Output} (right) window of LAMMPS-GUI after performing
+  the minimization simulation of \hyperref[lennard-jones-label]{Tutorial 1}.}
+\label{fig:chart-log}
+\end{figure}
+
+As the simulation launches, two additional windows will show up as shown
+in figure \ref{fig:chart-log}: a \textit{Charts} window showing the
+evolution of the total energy and an \textit{Output} window with the
+output LAMMPS writes to the console during a simulation. This includes
+the thermodynamic data in text form that is plotted in the \textit{Chart} window.
+
+If the plot in your \textit{Chart} window is missing data, then your
+computer is running the simulation too fast for LAMMPS--GUI to collect
+all the data.  You can adjust the settings in the
+\textit{Edit~$\to$~Preferences} dialog to collect data more frequently
+by reducing the \textit{GUI update interval} value.  At values lower
+than 10ms the time the GUI spends on collecting the data becomes
+significant and thus those should only be used for fast tutorials like
+this one.  You can also make your simulation slower by selecting
+\textit{None} in the \textit{Accelerators} tab.
+
+The potential energy can be seen to decrease from a positive value to a
+negative value.  The initially positive value of the potential energy is
+expected because the atoms have been created at random positions within
+the simulation box and some of them are very close, resulting in a large
+initial potential energy due to the repulsive branch of the
+Lennard-Jones potential.  As the energy minimization progresses, the
+energy decreases -- first rapidly -- then more slowly and plateaus at
+a negative value, indicating that the atoms have been displaced to
+reasonable distances from each other.
+
+Since the \textit{thermo\_style} command also included the output of the
```

Comment:
writing "ed" instead of "es" is a common typo I do. I suspect this is due to having never properly learned to type "blind". It happens because "d" is right next to "s" on my keyboard. ;-)